scripts/utils/ScriptPaths.mjs

```
@@ -0,0 +1,27 @@
+  static async updateManifest() {
+    return path.join(await this.scriptsDir(), 'update-manifest.sh');
+  }
+
+  static async listApexTests() {
+    return path.join(await this.scriptsDir(), 'list-apex-tests.mjs');
+  }
+
+  static async apexTestOperator() {
+    return path.join(await this.scriptsDir(), 'apex-test-operator.sh');
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)